package.json

```diff
@@ -20,6 +20,7 @@
 		"@tanstack/react-query-devtools": "^5.24.1",
 		"axios": "^1.6.7",
 		"fast-deep-equal": "^3.1.3",
+		"moment-timezone": "^0.5.45",
```

Comment:
Let's not use `moment` and `moment-timezone` as they are no longer maintained. Please use `date-fns` if you need to pull in date utilities. We may want to roll our own utilities (in the new date service!) if they are just date comparison and date math calls. Up to you.

Reply:
Removed in 0ac6fed